plugin.go

```diff
@@ -324,20 +324,35 @@ func handlerCreateEndpoint(w http.ResponseWriter, r *http.Request) {
 	 * from the network plugin to the runtime
 	 */
 	cmd = "ip"
-	args = []string{"link", "add", vhost_port, "type", "dummy"}
+	args = []string{"link", "add", vhostPort, "type", "dummy"}
 	if err := exec.Command(cmd, args...).Run(); err != nil {
 		resp.Err = fmt.Sprintf("Error EndPointCreate: [%v] [%v] [%v]",
 			cmd, args, err)
 		sendResponse(resp, w)
 		return
 	}
 
+	// Create a symlink on the filesystem to point to the endpoint
+	socketpath := fmt.Sprintf("/tmp/vhostuser_%s", ip)
+	err = os.Mkdir(socketpath, 0777)
```

Comment:
just a nit. Do we need 777 does 500 work?

Reply:
yes.